src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react"
+import getArticles from "../../api";
+import ArticleCard from "./ArticleCard";
+
+function ArticlesList(props) {
+
+const { currentArticles, setCurrentArticles, sortBy, setSortBy } = props;
+
+useEffect(() => {
+    getArticles().then((articles) => {
+        console.log(articles);
+        setCurrentArticles(articles);
+    });
+  }, []);
```

Comment:
- Nice 👍 
- Please remove console logs when submitting PRs
- Could you have created the currentArticles state in here? did you need to pass it down?